cmd/license-update.go

```diff
@@ -109,6 +109,10 @@ func performLicenseUpdate(licFile string, alias string, airgap bool) licUpdateMe
 		fatalIf(errDummy().Trace(), fmt.Sprintf("License has expired on %s", li.ExpiresAt))
 	}
 
+	if li.DeploymentID != getAdminInfo(alias).DeploymentID {
+		fatalIf(errDummy().Trace(), fmt.Sprintf("Invalid license"))
```

Comment:
```suggestion
		fatalIf(errDummy().Trace(), fmt.Sprintf("Invalid license"))
```

should be we more prescriptive about the error on why the license is invalid here?

Reply:
Isn't it better to keep credentials related error messages vague for security reasons? Providing information when license has expired is fine, but if it's not a valid license for the deployment, I feel it's better to just say it's invalid.

Reply:
Yeah but I was thinking we should say something like

```
license is invalid for the deployment '%s' 
```

Is what I would expect as an error, %s is just the alias name.

Reply:
sure, will do